diff --git a/tools/server/server-http.cpp b/tools/server/server-http.cpp
index 2ec137aa0..017694987 100644
--- a/tools/server/server-http.cpp
+++ b/tools/server/server-http.cpp
@@ -198,6 +198,7 @@ bool server_http_context::init(const common_params & params) {
         std::unordered_set<std::string> endpoints {
             "/health",
             "/v1/health",
+            "/metrics",
         };
         endpoints.insert(frontend_paths.begin(), frontend_paths.end());
         return endpoints;
